Cargo.toml

```diff
@@ -122,6 +122,8 @@ vlog = "0.1.4"
 twoway = "0.2.0"
 ordered-float = "1.0"
 nom = "5.0.0-beta1"
+openssl = { version = "0.10.23", features = ["vendored"] }
```

Comment:
why are we using replace, instead of directly specify libre-ssl as the dependency?